thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/tools/LoadMetricsFromPinotTool.java

```diff
@@ -117,6 +118,8 @@ private void initDAOs(File persistenceFile) {
         .getInstance(com.linkedin.thirdeye.datalayer.bao.jdbc.MetricConfigManagerImpl.class);
     dashboardConfigDAO = DaoProviderUtil
         .getInstance(com.linkedin.thirdeye.datalayer.bao.jdbc.DashboardConfigManagerImpl.class);
+    DAORegistry DAO_REGISTRY = DAORegistry.getInstance();
+    DAO_REGISTRY.registerDAOs(anomalyFunctionDAO, null, null, null, null, null, datasetConfigDAO, metricConfigDAO, dashboardConfigDAO);
```

Comment:
can we make it name based DAO_REGISTRY.register("anomalyFunctionDAO", anomalyFunctionDAO);